microsoftgraph/microsoft-graph-explorer-v4
# Feature: resizable components

## Overview
Enables the resizing of the response and request components so that a user has the ability to increase the size of the views. 
Increasing the size of the request component automatically increases the size of the response view

Closes #278 
Fixes #630 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Hover over the bottom border of the request area
* Notice the presence of the resize handles
* Resize the component up or down.
